Project.toml

```diff
@@ -21,28 +24,29 @@ SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 StrideArrays = "d1fa6d79-ef01-42a6-86c9-f7c551f8593b"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
 
 [compat]
 ACEbase = "0.4.2"
 Adapt = "3.5"
+BenchmarkTools = "1"
 ChainRulesCore = "1"
 Combinatorics = "1"
-BenchmarkTools = "1"
 ForwardDiff = "0.10"
 LoopVectorization = "0.12"
 LuxCore = "0.1.3"
 ObjectPools = "0.2.1"
 QuadGK = "2"
 SpecialFunctions = "2.2"
 StaticArrays = "1.5"
-julia = "1.8"
 StrideArrays = "0.1.25"
-
+julia = "1.8"
 
 [extras]
 Lux = "b2108857-7c20-44ae-9111-449ecde12c47"
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+ACEcore = "44c1e890-45d1-48ea-94d6-c2ea5b573f71"
```

Comment:
I assume this is a temporary dependency and we will remove it with the next version?

Reply:
Yes. Test should only be used in testing and ACEcore should be removed entired.

Reply:
News only talking about ACEcore